### PR TITLE
Fix func receiver typo

### PR DESCRIPTION
**Pull request by nathj07**
Apologies for this one, it a daft mistake made on my last PR. 

I had accidentally used `x` as the function receiver rather than `xl`. I think this is worth merging before a release.

Ping @jamesmarchant for review once again, thanks

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -1,4 +1,4 @@
-module github.com/thedatashed/xlsxreader
+module github.com/nathj07/xlsxreader
```
Comment (jamesmarchant) [inline go.mod:1]: This should still be `thedatashed`

File: test/main.go [hunk 2]
```diff
@@ -3,7 +3,7 @@ package main
 import (
 	"fmt"
 
-	"github.com/thedatashed/xlsxreader"
+	"github.com/nathj07/xlsxreader"
```
Comment (jamesmarchant) [inline test/main.go:6]: same here - This should still be `thedatashed`

**Review by jamesmarchant — CHANGES_REQUESTED** [review]

File: test/main.go (hunk 2 above)
Comment (nathj07) [reply test/main.go:6]: Apologies, it should be that indeed. Let me fix this up

**Comment by nathj07** [comment]
@jamesmarchant apologies for that confusion, should be sorted now. Thanks

**Review by jamesmarchant — APPROVED** [review]

**Pull request merged**